features/tests/test_api.py

```diff
@@ -341,6 +343,35 @@ def test_feature_opening_hours(snapshot, api_client):
     snapshot.assert_match(executed)
 
 
+def test_price_list(snapshot, api_client):
+    PriceTagFactory(price=Decimal("100.01"), unit="a month")
+    PriceTagFactory(price=Decimal("200.01"), unit="a year")
```

Comment:
I guess, you could just do `PriceTagFactory.create_batch(2)` if you just need two objects in the DB and you do not care for / do not use their attributes.

Reply:
In this particular case I care that the attributes have the same values for the sake of snapshot testing.

Reply:
alright! 👍